src/msg/async/AsyncConnection.cc

```diff
@@ -443,7 +449,14 @@ void AsyncConnection::process() {
     case STATE_ACCEPTING: {
       center->create_file_event(cs.fd(), EVENT_READABLE, read_handler);
       state = STATE_CONNECTION_ESTABLISHED;
-
+      if (async_msgr->cct->_conf->mon_use_min_delay_socket) {
+        if (async_msgr->get_mytype() == CEPH_ENTITY_TYPE_MON &&
+            peer_is_mon()) {
+          cs.set_priority(cs.fd(),
+                          SOCKET_PRIORITY_MIN_DELAY,
```

Comment:
Nit: no need to line break between `cs.fd()` and `SOCKET_PRIORITY_MIN_DELAY`.

Reply:
done